lib/tasks/fastly.rake

```diff
@@ -1,19 +1,21 @@
 namespace :fastly do
   desc "Update Fastly configs"
   task update_configs: :environment do
-    fastly_credentials = %w[
-      FASTLY_API_KEY
-      FASTLY_SERVICE_ID
-    ]
+    unless ENV["SKIP_FASTLY_CONFIG_UPDATE"] == "true"
```

Comment:
we may want to document that this variable can be used to skip through fastly

Reply:
Good call, I will throw it in the on-call docs.